src/org/labkey/remoteapi/issues/IssueModel.java

```diff
@@ -0,0 +1,213 @@
+package org.labkey.remoteapi.issues;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.labkey.api.collections.CaseInsensitiveHashMap;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class IssueModel
+{
+    // keys
+    private final String TITLE= "title";
+    private final String ISSUE_ID = "issueid";
+    private final String ISSUE_DEF_NAME = "issueDefName";
+    private final String ASSIGNED_TO = "assignedto";
+    private final String ACTION = "action";
+    private final String TYPE = "type";
+    private final String PRIORITY = "priority";
+    private final String COMMENT = "comment";
+    private final String NOTIFY_LIST = "notifyList";
+    private final String RESOLUTION = "resolution";
+    private final String RESOLVED = "resolved";
+    private final String STATUS = "status";
```

Comment:
sometimes `enums` are easier to use than `String` constants

Reply:
I'm happy to put these into an enum!